api/src/main/java/org/openmrs/module/coreapps/CoreAppsConstants.java

```diff
@@ -53,4 +53,5 @@
 
    // Condition list
    public static final String MANAGE_CONDITIONS_PRIVILEGE = "Task: Manage Condition Lists";
+   public static final String CONDITIONS_TO_BE_DISPLAYED = "conditionList.class";
```

Comment:
Can you take a look at the naming convention for global property constants? https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/util/OpenmrsConstants.java